test/libsinsp_e2e/container/container.cpp

```diff
@@ -131,7 +131,8 @@ TEST_F(sys_call_test, container_cgroups)
 
 static int clone_callback(void* arg)
 {
-	sleep(5);
+	sleep(1);
```

Comment:
Can we leave at least a note/comment here? This feels so weird...

Reply:
Just pushed! As I said in the comment, this behaviour needs more investigation and I'll take of it in the following PRs.

Reply:
yeah we can even merge the PR with these changes this but I think we need to fix this ASAP because `sleep` can introduce a lot of entropy, even more, if we consider that we run these tests in little CI machines with few CPUs